kernel/cond.c

```diff
@@ -34,7 +34,7 @@ int myst_cond_destroy(myst_cond_t* c)
     myst_spin_lock(&c->lock);
 
     /* Fail if queue is not empty */
-    if (c->queue.front)
+    if (!myst_thread_queue_empty(&c->queue))
```

Comment:
I would like to remove this accessor since the only consumer of this field is the condition itself.

Reply:
Reverted.